ludwig/utils/misc_utils.py

```diff
@@ -111,28 +105,18 @@ def set_default_values(dictionary, default_value_dictionary):
 
 
 def get_class_attributes(c):
-    return set(
-        i for i in dir(c)
-        if not callable(getattr(c, i)) and not i.startswith("_")
-    )
+    return set(i for i in dir(c) if not callable(getattr(c, i)) and not i.startswith("_"))
 
 
-def get_output_directory(
-        output_directory,
-        experiment_name,
-        model_name='run'
-):
-    base_dir_name = os.path.join(
-        output_directory,
-        experiment_name + ('_' if model_name else '') + model_name
-    )
-    return find_non_existing_dir_by_adding_suffix(base_dir_name)
+def get_output_directory(output_directory, experiment_name, model_name='run'):
+    base_dir_name = os.path.join(output_directory,
+                                 experiment_name + ('_' if model_name else '') + model_name)
```

Comment:
[Nit], if `model_name` is `None` it could result in TypeError.

Maybe `experiment_name + ('_' + model_name if model_name else '')`

Reply:
Thanks! Updated